src/main/scala/com/stratio/receiver/RabbitMQInputDStream.scala

```diff
@@ -98,13 +98,14 @@ class RabbitMQReceiver(params: Map[String, String], storageLevel: StorageLevel)
       log.info("RabbitMQ Input waiting for messages")
       val consumer: QueueingConsumer = new QueueingConsumer(channel)
       log.info("start consuming data")
-      channel.basicConsume(queueName, true, consumer)
+      channel.basicConsume(queueName, false, consumer)
```

Comment:
Please, set extract this boolean as a parameter.
Thank you for your contribution!